rhc-ose-ansible/roles/disconnected-git/defaults/main.yaml

```diff
@@ -0,0 +1,15 @@
+---
+git_repo_home: "/opt/git"
+ose_git_repo_home: "{{ git_repo_home }}/openshift"
+git_user: "git"
+
+# List of Authorized Keys to Add
+git_user_authorized_keys:
+
+ose_example_repos:
```

Comment:
@sabre1041 could we move this out to an inventory/group_vars/disconnected-git file so it becomes part of inventory rather than hardcoded in the role?